docs/src/main/asciidoc/security-webauthn.adoc

```diff
@@ -784,7 +720,9 @@ This returns a 204 with no body.
 
 === Trigger a login
 
-`POST /q/webauthn/callback`: Trigger a login
+This uses the two cookies set by the login challenge and clears them. It
+also uses your link:{webauthn-api}/io/quarkus/security/webauthn/WebAuthnUserProvider.html[`WebAuthnUserProvider`] to find and update the credentials, and sets up
```

Comment:
What kind of credential update is required during the login ?

Reply:
This is described in the docs previously, where an example implementation of the `WebAuthnUserProvider` class is given. I could repeat he explanation, but I'm not sure it's useful here.

Reply:
@FroMage OK